### PR TITLE
Fix Pet Cove Story Plaques

### PR DESCRIPTION
**Pull request by zxc8027**
For a school assignment, I decided to look at the Pet Cove Story Plaques issue (#271). the bug is due to a special case that is not accounted for in the handler of the plaque being interacted with.

### CODE REVIEW

**Comment by teccheck** [comment]
Thank you 👍 The code looks good, I think we can merge it.

**Review by TheNexusAvenger — APPROVED** [review]
Wasn't expecting it to be that simple of a fix. If I knew it was that simple, I would have taken this last year.

~~I will test it within an hour to test it before merging.~~ Edit: Another maintainer beat me to testing.

**Review by Frozenreflex — APPROVED** [review]

**Comment by enteryournamehere** [comment]
Cool! Thank you for your contribution!

image

**Review by enteryournamehere — APPROVED** [review]

**Pull request merged**
